pkg/config/resources/exec/provider.go

```diff
@@ -61,23 +61,53 @@ func (p *Provider) Create(ctx context.Context) error {
 
 	p.log.Info("executing script", "ref", p.config.Meta.ID, "script", p.config.Script)
 
+	outPath := fmt.Sprintf("%s/%s.out", os.TempDir(), p.config.Meta.ID)
```

Comment:
Just for future reference, should we write these temp files to os temp or jumppad temp?

Reply:
I saw we had jp temp after creating this, I might change it and push later.